hack/lib/serverless.bash

```diff
@@ -422,3 +422,37 @@ function gather_knative_state {
     "${IMAGE_OPTION[@]}" \
     --dest-dir "$gather_dir" > "${gather_dir}/gather-knative.log"
 }
+
+# delete serverless test resources
+function teardown_extras {
+  logger.warn "😭  Teardown serverless extras..."
+
+  # remove routes
+  logger.info 'Removing serverless test routes'
+  if oc get route -A | grep "metrics-eventing" >/dev/null 2>&1; then
+    oc delete --ignore-not-found=true route/metrics-eventing -n openshift-serverless
```

Comment:
As of now we have removed teardown_extras() function from serverless.bash.  We are not creating cluster every time we run the test suits. We use same already created cluster to run the tests. So if it works in serverless-release 1.25, then we can got with this and can remove this. Or else we may need to add few code for teardown the extra resources. And logically we think that it will not affect X86 code also.